.github/workflows/tests.yml

```diff
@@ -28,17 +28,27 @@ jobs:
       - name: Upgrade pip
         run: python -m pip install --upgrade pip
 
-      - name: Install dependencies
-        run: pip install ".[tests]"
+      - name: Cache Python packages
```

Comment:
I think it should be done with https://docs.astral.sh/uv/guides/integration/github/#caching

``` 
- name: Enable caching
  uses: astral-sh/setup-uv@v4
  with:
    enable-cache: true
  ```

Reply:
Nice, thank you, done